packages/components/psammead-oembed/README.md

```diff
@@ -0,0 +1,77 @@
+# ⛔️ This is an alpha component ⛔️
```

Comment:
It might be beneficial to mention the `DomPurifier` node module is what we're relying on to do our sanitisation.

Reply:
That's a nice idea. I've updated the readme to include a section on sanitisation.